pkg/roachpb/metadata.proto

```diff
@@ -287,6 +281,8 @@ message RangeDescriptor {
   // queue is enabled. With sticky_bit, users can manually split ranges without
   // disabling the merge queue.
   optional util.hlc.Timestamp sticky_bit = 7;
+
+  reserved 8;
```

Comment:
Some teams have a convention to [reserve](https://developers.google.com/protocol-buffers/docs/proto3#reserved) the field name as well. Do we have any convention?

Otherwise, this field name looks unlikely to be reused, so I would leave this as is :)

Reply:
Nah, all of our `reserved` declarations use IDs only.